### ISSUE TITLE
Upload fails, no options to resolve file upload issue

### ISSUE THREAD

**Issue by gspeed0689** [zenodo/zenodo#2481]
Browser & OS (see also https://www.whatismybrowser.com/): Edge 117 (Chromium based):Windows 10

## Describe the bug
I am making an update to one one of my repositories, and some files failed to upload. They currently show as pending in the file list, with no option to delete and retry the upload. A screenshot of my current view is included below.

## Steps to Reproduce
1. Go to create a new version of an existing repository
2. Drag and drop new files to upload to Zenodo
3. Unknown issue causes upload to fail
4. No options to adjust, delete, or re-upload files. 

## Screenshots (if applicable)
![image](https://github.com/zenodo/zenodo/assets/6378547/c5de52dc-c139-46e7-85f1-34ce426850a6)

**Comment by JGuetschow** [comment]
I can confirm the Issue for Firefox 117 on Linux (Mint 21.2)

**Comment by peterjc** [comment]
Confirming with Firefox 118 or Chrome 117 on macOS (Monterey). In my case this is a draft submission not yet published:

<img width="730" alt="Screenshot 2023-10-16 at 13 36 34" src="https://github.com/zenodo/zenodo/assets/63959/94d8f2ef-3ae5-4729-ba22-61a1bfbb2925">

Attempting to re-upload the pending file fails ("The following files already exist ..."):
<img width="1139" alt="Screenshot 2023-10-16 at 13 37 25" src="https://github.com/zenodo/zenodo/assets/63959/ef890a4a-b334-4d26-b7e2-9a7e005eeb85">

**Comment by peterjc** [comment]
Confirmed, I now see a delete button on the stuck pending file, and so can delete it and retry the upload. Thanks.

**Comment by gspeed0689** [comment]
I have delete buttons as well, thank you for the quick fix!

**Comment by jrfep** [comment]
Yes, I can also confirm the delete button works now.

**Comment by Raen111** [comment]
вт, 24 окт. 2023 г., 01:37 José R. Ferrer Paris ***@***.***>:

**Comment by Raen11** [comment]
Пидарасы как вы тут,хуесоски

**Comment by Raen11** [comment]
> Да, я также могу подтвердить, что кнопка удаления теперь работает.

Пойди хуесос отсоси у соседа по брацки так ты говорил,гомик фууу

**Comment by peterjc** [comment]
@gspeed0689 you can probably close this issue now. I don't know if as the issue opener you can flag the above as spam or remove it, we might need someone from Zenodo to do that? I have reported the user (a new account) to GitHub via the option on their profile page.

**Comment by kikislater** [comment]
> Yes, I can also confirm the delete button works now.

Yes but upload (it seems above 100mo) still fail ... Nice there is this option but it still a platform break !

**Comment by kikislater** [comment]
100mo is now ok, 1.8GB ok, 3.4GB NOK using API =>
SSLError: HTTPSConnectionPool(host='zenodo.org', port=443): Max retries exceeded with url: /api/files
```
/xxxxxxx-6cb9-43b9-9fd2-xxxxxxxxxxxxx/DCIM.zip?access_token=xxxxxxxxxxxxx (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:2426)')))
```

8GB ok through webgui

**Comment by peterjc** [comment]
@jarredou can you delete the failed upload, and try again (as described in the issue comments)?

**Comment by kikislater** [comment]
Works for me, I ended with a more complex python script in a loop which verify checksum on server. if checksum is bad, remove file and try again. 

Details could be seen here : 

https://dronemd.readthedocs.io/en/latest/How_to_upload_to_zenodo.html#Upload-to-Zenodo

**Comment by kwiechen** [comment]
this problem is not resolved at all - uploading files stops with message pending. Delete and re-upload does not work

**Comment by gspeed0689** [comment]
@kwiechen this morning I am (probably) experiencing the upload error you are, however I believe this error is different from the upload fail issue I originally opened this ticket for. Currently you can remove and retry the upload, when I opened the ticket that was not possible. 

I think Zenodo has some sort of outage this morning. 

Edit: 

![Image](https://github.com/user-attachments/assets/343e85a5-55a0-42a4-b221-a036b539fd18)

**Comment by LoponteHF** [comment]
Dear all,

I have a draft up on Zenodo that I'm trying to publish, but it's always throwing me an error:

![Image](https://github.com/user-attachments/assets/acff0888-5d67-4450-bd79-0c49d582d4e9)

I think this is related to this issue because I have 4 "ghost" files that I can't remove from my draft, no matter what I do... Even if I click the trashbin icon, the upload disappears, I save the draft and when I come back or reload the page, it is back again:

![Image](https://github.com/user-attachments/assets/dd2ab8ee-d654-4ee2-949e-bd671e7350f2)

How can I solve this? I sent a message to support on the website but I get no replies or any indication that I might get one.

Thanks in advance.

ps.: Chrome 132 on Windows 11, tried Firefox and Edge as well, to no avail.

**Comment by CeresBarros** [comment]
I'm having a similar issue as well. Three of my files never finish uploading. Some are larger than others already in the repository, others aren't. I'm well below the total storage for this project. 

I have tried removing the pending uploads and re-trying multiple times. I've also tried to change the storage space. 

The only thing that seems to be working is uploading one at file at  a time. At least I've gotten further today by only uploading one of the files (still uploading though, so we'll see!).

Final note, I'm uploading .zip files.

Update: uploading one file at a time is working for me
